recipes/main/thai_red_curry.md

```diff
@@ -0,0 +1,41 @@
+# Thai red curry
+
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-30%25-green.svg)
+![Speed Badge](https://img.shields.io/badge/Speed-1.5hr-orange.svg)
+![Reception Badge](https://img.shields.io/badge/Reception-Positive-green.svg)
+![Cost Badge](https://img.shields.io/badge/Cost-Average-yellow.svg)
+![Vegetarian Friendly Badge](https://img.shields.io/badge/Vegetarian-True-brightgreen.svg)
+![Vegan Friendly Badge](https://img.shields.io/badge/Vegan-True-brightgreen.svg)
+
+**Head chef:** [@matthewdking](https://github.com/matthewdking/)
+
+**Sous chef:** [@Karyum](https://github.com/Karyum/)
```

Comment:
i only peeled potatoes but i'll take the credit :sweat_smile:

Reply:
Every little helps man :+1: